src/test/regress/sql/gp_tsrf.sql

```diff
@@ -62,3 +62,77 @@ SELECT * FROM t_outer WHERE t_outer.b IN (SELECT generate_series(1, t_outer.b) F
 EXPLAIN (VERBOSE, COSTS OFF)
   SELECT * FROM t_outer WHERE t_outer.b IN (SELECT generate_series(1, t_outer.b)  FROM t_inner);
 DROP TABLE t_outer, t_inner;
+
+-- Check for proper resource deallocation for SRF which has been squelched
+
+-- start_ignore
+DROP TABLE IF EXISTS ao1_srf_test;
```

Comment:
use lowercase as in the rest of the file. or use uppercase everywhere as it is used in one part of the file.